site/src/atomics/spacing.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Spacing Atomics
+description: Spacing related atomic css classes for the Atlas Design System
+template: standard
+---
+
```

Comment:
After reading through this entire page, I think that we should try to stress the pattern. We've missed out on two combinations because of this.

`<cssprop>-<size>-<screensize>`

Properties: `margin, padding, padding|margin-left|right|top|bottom, padding-inline, padding-block`
Sizes: xs, s, m, l, xl, xxl, xxxl

Reply:
Updated